cider/exec/template/DateTimeTranslator.cpp

```diff
@@ -115,8 +115,39 @@ std::string from_datetrunc_field(const DatetruncField& fieldno) {
   return "";
 }
 
+static const std::map<std::string, ExtractField> presto_extract_function_lookup = {
+    {"year", kYEAR},
+    {"quarter", kQUARTER},
+    {"month", kMONTH},
+    {"day", kDAY},
+    {"quarterday", kQUARTERDAY},
+    {"hour", kHOUR},
+    {"minute", kMINUTE},
+    {"second", kSECOND},
+    {"millisecond", kMILLISECOND},
+    {"microsecond", kMICROSECOND},
+    {"nanosecond", kNANOSECOND},
+    {"day_of_week", kDOW},
+    {"isodow", kISODOW},
+    {"day_of_year", kDOY},
+    {"epoch", kEPOCH},
+    {"week", kWEEK},
+    {"week_sunday", kWEEK_SUNDAY},
+    {"week_saturday", kWEEK_SATURDAY},
+    {"dateepoch", kDATEEPOCH}};
+
```

Comment:
remove empty line?

Reply:
next line is 

```}  // namespace```

should there be no extra line?